src/components/TimeRelated.tsx

```diff
@@ -11,10 +11,9 @@ export interface IProps {
 }
 
 const SetHours = (timeData: string) => {
-    const currentDate = new Date();
-    const inputDate = new Date(timeData);
-
-    return isSameDay(inputDate, currentDate) ? 'Сейчас' : format(new Date(timeData), 'HH:mm');
```

Comment:
You could use this code and just change isSameDay to isSameHour. Also use inputDate in format instead of creating new instance of the same object